projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -89,6 +91,26 @@ public static function init() {
 		do_action( 'jetpack_mu_wpcom_initialized' );
 	}
 
+	/**
+	 * Fixes translation file path MD5 mismatches caused by vendor -> jetpack_vendor migration.
+	 *
+	 * WordPress generates the translation file's MD5 hash based on the script's relative path.
+	 * Since the path structure changed, we rewrite `jetpack_vendor/` back to `vendor/` before hashing.
+	 *
+	 * @param string|false $relative The relative script path (before MD5 hashing).
+	 * @param string       $src     The script's original source path (unused).
+	 * @return string|false Updated relative path that keeps WordPress's MD5 hash consistent.
+	 */
+	public static function fix_translation_relative_path_mismatch_wpcom_jetpack( $relative, $src ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+		// Ensure $relative is valid and begins with "jetpack_vendor/"
+		if ( $relative && str_contains( $relative, 'jetpack_vendor/automattic/jetpack-mu-wpcom/src/build/' ) ) {
```

Comment:
The comment says "begins with", but the code does "contains"?

Reply:
Ah, I see 1bf4c78c8 explains the difference. Another option might be to adjust the priority on the hook here so it comes after the one from wpcom's `0-wpcom-jetpack-loader.php` that strips out the sun/moon.